composer.json

```diff
@@ -128,7 +128,7 @@
         "drupal/views_bulk_operations": "^3.6",
         "drupal/xmlsitemap": "~1.0",
         "ext-imagick": "*",
-        "su-soe/soe_basic": "^8.1.1",
+        "su-soe/soe_basic": "~8.1.0",
```

Comment:
Why the `~` instead of the `^`

Reply:
To match the usage in the rest of the file.  Personally, I think the caret is better, but I went for consistency.

Reply:
We normally go with the caret. Its less strict. for this situation it really doesn't matter since the theme isn't `8.2.0` or anything

Reply:
Next release, I'll update all of them.  Carets are the way to go.